index.html

```diff
@@ -54,5 +55,148 @@ <h1>I’m Fadahunsi Glad to see you!</h1>
+              </ul>
+              <button class="view-work-btn">See Project</button>
+            </div>
+          </section>
+          <section id="aboutMe" class="about-me">
+            <div class="top">
+              <h2 class="title">About Myself</h2>
+              <p class="introduction">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <div class="connect">
+                    <a>LET'S CONNECT</a>
+                </div>
+                <div class="header-section-icon-container">
+                    <ul>
+                        <li>
+                        <a href="/">
+                            <img src="./img/Linkedin.png" alt="The icons" />
+                        </a>
+                    </li>
+                        <li>
+                        <a href="/">
+                            <img src="./img/angellist.png" alt="The icons" />
+                        </a>
+                    </li>
+                        <li>
+                        <a href="/">
+                            <img src="./img/github.png" alt="The icons" />
+                        </a>
+                    </li>
+                        <li>
+                        <a href="/">
+                            <img src="./img/medium.png" alt="The icons" />
+                        </a>
+                    </li>
+                        <li>
+                        <a href="/">
+                            <img src="./img/twitter.png" alt="The icons" />
+                        </a>
+                    </li>
+                    </ul>
+                </div>
+      
+              <button class="resume-btn">Get my resume</button>
+            </div>
+            <div class="bottom">
+              <ul class="technical-knowledge">
+                <li>
+                  <div class="knowledge-area active">
+                    <h3 class="title">Languages</h3>
+                    <i class="fa fa-chevron-down"></i>
+                  </div>
+                  <ul class="technical-knowledge-list">
+                    <li><img src="./img/technology/javascript.png" alt="javascript">JavaScript</li>
+                    <li><img src="./img/technology/css.png" alt="html">HTML</li>
+                    <li><img src="./img/technology/html.png" alt="css">CSS</li>
+                  </ul>
+                </li>
+                <li>
+                  <div class="knowledge-area">
+                    <h3 class="title">Frameworks</h3>
+                    <i class="fa fa-chevron-right"></i>
+                  </div>
+                </li>
+                <li>
+                  <div class="knowledge-area">
+                    <h3 class="title">Skills</h3>
+                    <i class="fa fa-chevron-right"></i>
+                  </div>
```

Comment:
- [ ] Same here please try to use the right font the one provided in the Figma design 
![image](https://user-images.githubusercontent.com/61790587/170544838-53cc0916-c07a-479f-9f06-76adf6f50c02.png)